api/src/pcapi/routes/serialization/offerers_serialize.py

```diff
@@ -93,11 +92,7 @@ class GetOffererResponseModel(BaseModel):
 
     @classmethod
     # type: ignore
-    def from_orm(
-        cls,
-        offerer: offerers_models.Offerer,
-        venue_stats_by_ids: dict[int, venues_serialize.VenueStatsResponseModel] | None = None,
-    ):
+    def from_orm(cls, offerer: offerers_models.Offerer):
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** pour retirer le ` # type: ignore`
```suggestion
    def from_orm(cls: Any, offerer: offerers_models.Offerer) -> "GetOffererResponseModel":
```

Reply:
Noté, je vais faire ça dans une autre branche en cours où je supprime quelques "type: ignore". Merci !